build.gradle

```diff
@@ -240,13 +240,14 @@ subprojects {
             targetCompatibility = 1.8
         }
 
-        tasks.withType(AbstractCompile).all {
+        tasks.matching { it instanceof JavaCompile || it instanceof GroovyCompile }.all {
```

Comment:
The `kotlin` plugin contributes `KotlinCompile` which extends `AbstractCompile` but doesn't have attributes like `options`.